Makefile

```diff
@@ -1,7 +1,14 @@
-default: build
+# Parse -j from MAKEFLAGS for parallelism.
+GERBIL_BUILD_CORES := $(or $(patsubst -j%,%,$(filter -j%,$(MAKEFLAGS))),1) 
 
-build:
-	./build.sh
+all: build
+
+build-env.sh:
+	make clean
```

Comment:
actually this might fail one empty checkout, configure fetches the submodule.
So lets remove.

Reply:
just the clean i mean.